packages/cli/src/active-executions.ts

```diff
@@ -61,9 +61,7 @@ export class ActiveExecutions {
 				workflowId: executionData.workflowData.id,
 			};
 
-			if (executionData.retryOf !== undefined) {
-				fullExecutionData.retryOf = executionData.retryOf.toString();
-			}
+			fullExecutionData.retryOf = executionData.retryOf ?? undefined;
```

Comment:
According to the type `?? undefined` is not necessary.

![Capture 2025-02-03 at 14 10 51@2x](https://github.com/user-attachments/assets/709df9f0-620b-426f-8b29-10567ad7dc2b)

Same elsewhere.

Reply:
`retryOf` typings are still not 100% reliable, and sometimes this is `null` because the value came from typeorm.